urdf_to_simox_xml/src/urdf_to_simox_xml.cpp

```diff
@@ -23,7 +23,6 @@
 #include <string>
 #include <sstream>
 #include <vector>
-#include <boost/foreach.hpp>
```

Comment:
why did you move that to the header? It's better to keep the dependencies in the header to a minimum.